src/schemathesis/engine/phases/unit/_executor.py

```diff
@@ -304,6 +309,37 @@ def scenario_finished(status: Status) -> events.ScenarioFinished:
     for error in deduplicate_errors(errors):
         yield non_fatal_error(error)
 
+    # Collect successful responses to use in subsequent test generation
+    # In the future, collecting unsuccessful responses also could be useful
+    # to understand if some generated data is always rejected
+    extra_data_source = (
+        ctx.extra_data_source
+        if isinstance(phase_config, FuzzingPhaseConfig) and phase_config.extra_data_sources.is_enabled
```

Comment:
It should be recorded in other phases anyway - results will be available for the fuzzing phase